Rakefile

```diff
@@ -57,7 +57,7 @@ task 'setup_libs' do
   jmx_version = `venv/bin/python -c "import config ; print config.JMX_VERSION"`
   jmx_version = jmx_version.delete("\n")
   puts "jmx-fetch version: #{jmx_version}"
-  jmx_artifact = "jmxfetch-#{jmx_version}-jar-with-dependencies.jar"
+  jmx_artifact = "jmxfetch-#{jmx_version}.jar"
```

Comment:
Should this stay as "*-jar-with-dependencies"? Looks like this version is published as part of the artifacts: https://repo1.maven.org/maven2/com/datadoghq/jmxfetch/0.47.9/

Same question for changes in packaging/datadog-agent/source/setup_agent.sh